### PR TITLE
fix(admin): bind log to configure it before using

### PR DESCRIPTION
**Pull request by dbanda**
The logger in `AuditLog` wasn't configured properly and was lazy so it was ignored in k8s. Adding bind to force it to configure properly and add the processors.

### CODE REVIEW

File: snuba/admin/audit_log/base.py [hunk 1]
```diff
@@ -10,7 +10,9 @@
 
 class AuditLog:
     def __init__(self) -> None:
-        self.logger = structlog.get_logger()
+        self.logger = structlog.get_logger().bind(
```
Comment (volokluev) [inline snuba/admin/audit_log/base.py:13]: would be good to explain why the bind is important in a comment

**Review by volokluev — APPROVED** [review]

File: snuba/admin/audit_log/base.py [hunk 2]
```diff
@@ -10,7 +10,9 @@
...
+            module=self.__class__.__module__, context_class=self.__class__.__qualname__
```
Comment (MeredithAnya) [inline snuba/admin/audit_log/base.py:14]: what does the `context_class` give us? is the module not enough?

**Comment by MeredithAnya** [comment]
Also, from reading the docs, it looks like we can just call `bind` without any parameters ( in the "important" note in https://www.structlog.org/en/stable/configuration.html#configuration). Is the problem that we didn't call `bind`? or that we didn't call `bind` with the parameters you added?

File: snuba/admin/audit_log/base.py (hunk 2 above)
Comment (dbanda) [reply snuba/admin/audit_log/base.py:14]: module is enough. anything to trigger the binding would work

File: snuba/admin/audit_log/base.py (hunk 1 above)
Comment (dbanda) [reply snuba/admin/audit_log/base.py:15]: added

**Pull request merged**